.github/workflows/test.yml

```diff
@@ -375,11 +373,12 @@ jobs:
         with:
           path: ~/.pub-cache/
           key: ${{ runner.os }}-pub-cache-${{ hashFiles('frostsnapp/pubspec.lock') }}-v1
+          restore-keys: |
+            ${{ runner.os }}-pub-cache-
 
       - uses: subosito/flutter-action@v2
         with:
           channel: stable
-          cache: true
```

Comment:
i think this looks for **/pubspec.lock, which doesn't work? Will revisit tomorrow
https://github.com/frostsnap/frostsnap/actions/runs/11666583441/job/32481995066

Reply:
we can just commit `pubspec.lock`. It would be better to use the flutter action caching approach.

Reply:
Was already commited. I just undid this change and it magically worked. I also deleted all the caches, ran again, and it still worked.